apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -19,7 +19,7 @@ def restart_url
   # @param url [#to_s, nil] url to access
   # @param role [String] app role i.e. "vdi", "shell", etc.
   # @return nil if url not set or the HTML string for the bootstrap nav link
-  def nav_link(title, icon, url, target: "_self", role: nil)
+  def nav_link(title, icon, url, target: "", role: nil)
```

Comment:
Why change "_self" to ""? We are still setting target here, I'm not sure having `<a target=""` is probably not desirable.

Reply:
Ah, so this was probably set when (if we ever intended to?) use https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base with target=_blank. Since that is not the case, explicitly setting `_self` probably is not necessary. But in the case of the link arguments, I think we would omit the target from passing if it is an empty string - unless the Rails helper knows to ignore that when building the html.